frontend/public/components/_list.scss

```diff
@@ -56,3 +56,38 @@
   right: 0;
   bottom: 0;
 }
+
+/* Make resource list (co-m-table-grid) scrollable
+--------------------------------------------------- */
+
+@media(max-width: $grid-float-breakpoint) {
+  .table-grid-wrapper {
+    overflow: hidden;
+    @include scroll-shadows-horizontal-other();
+  }
+  .co-m-table-grid--responsive {
```

Comment:
Wondering, to stick with BEM, if we can flatten this somehow & get the `.row` out of the child position.

Since it is overriding `.row`, maybe there is a better approach.  Could we conditionally apply class `row` in the `jsx`, and when the table parent is `co-m-table-grid--responsive` make the child something like `table-row--responsive` as well (and not need to override bootstrap?)

I do tend to dislike the `&--` modifiers as well, as it deeply couples parent-child-grandchild relationships in CSS, something we would definitely try to avoid in JS.  I haven't spent a ton of time thinking about ways to get away from it, but I think it is antithetical to the BEM approach as well.

I wouldn't necessarily block the PR over this, but wanted to bring it up early while we are still setting a trajectory on our CSS practices in this new console.